.github/workflows/ci.yaml

```
@@ -174,84 +325,74 @@ jobs:
 
   publish-docs:
     name: Publish documentation website to GitHub pages
-    needs: ["nox", "build", "protolint"]
+    needs: ["nox-all", "nox-cross-arch-all", "test-installation", "protolint"]
```

This is amr64-related, it should be changed.
```suggestion
    needs: ["nox-all", "test-installation", "protolint"]
```